install_magenta.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Check which operating system
+if [ "$(uname)" == "Darwin" ]; then
+    echo "Mac OS Detected"
+    export OS="MAC"
+    export MINICONDA_SCRIPT=Miniconda2-latest-MacOSX-x86_64.sh
+    # Mac OS X, CPU only, Python 2.7:
+    export TF_BINARY_URL=https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-0.11.0rc0-py2-none-any.whl
```

Comment:
We'll want to add a note to our release process to make sure we update these URLs to match our setup.py version requirements.